.github/workflows/push-treasurenet-mainnet-artifacts-to-s3.yml

```
@@ -0,0 +1,178 @@
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b
+        with:
+          fetch-depth: 0
+
+      - name: Override build command from manual input
+        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.build_target != '' }}
+        run: |
+          set -euo pipefail
+          echo "BUILD_COMMAND=${{ github.event.inputs.build_target }}" >> $GITHUB_ENV
+
+      - name: Set up Go environment
+        uses: actions/setup-go@v3
```

actionlint (1.7.8)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)